packages/fxa-admin-server/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "react-app",
```

Comment:
I might be missing something but I don't think this should extend `react-app` since there's not any React in the admin-server (just the admin-panel). The auth-server extends `plugin:fxa/server`, maybe we use that?

Reply:
OK, I think this will work for linting TypeScript w/ a similar config to react-app and react-app/jest (minus the React bits):

```json
{
  "env": {
    "es6": true,
    "node": true
  },
  "plugins": ["import"],
  "extends": ["eslint:recommended", "plugin:import/recommended"],
  "rules": {
    "import/no-unresolved": "off"
  },
  "root": true,
  "ignorePatterns": ["dist"],
  "overrides": [
    {
      "files": ["**/*.ts?(x)"],
      "parser": "@typescript-eslint/parser",
      "parserOptions": {
        "ecmaVersion": 2018,
        "sourceType": "module",
        "warnOnUnsupportedTypeScriptVersion": true
      },
      "plugins": ["@typescript-eslint"],
      "extends": ["plugin:@typescript-eslint/recommended"],
      "rules": {}
    },
    {
      "files": ["**/*.{spec,test}.*", "*.e2e-spec.*", "src/mocks.ts"],
      "env": {
        "jest/globals": true
      },
      "plugins": ["jest"],
      "extends": ["plugin:jest/recommended"],
      "rules": {}
    }
  ]
}
```